cmake/cpack.cmake

```diff
@@ -0,0 +1,5 @@
+SET(CPACK_PACKAGE_VERSION_MAJOR 0)
```

Comment:
recommend using git-versioning.cmake from e.g. acquire-video-runtime. That will set the package name based on the git tag.

Reply:
The reason I went this route was that I noticed that without setting `CPACK_PACKAGE_VERSION_*` in this file, all the package versions on the builds were automatically set to 0.1.1. But this must have been because there's no git-versioning.cmake in the driver repos (and no tags are set anyway). This way we can automate releases, too.